### PR TITLE
Renaming datasets

### PR DESCRIPTION
**Pull request by charyeezy**

### CODE REVIEW

**Comment by boba-and-beer** [comment]
@charyeezy I don't think it's necessary for us to force breaking changes here. 

I think it's better to have multiple assignments.  
For example: 
```
get_ecommerce_1_dataset = get_ecommerce_dataset_vectors = get_ecommerce_dataset
```

File: relevanceai/datasets.py [hunk 1]
```diff
@@ -480,6 +473,13 @@ def get_machine_learning_research_dataset():
     raise NotImplementedError
 
 
-get_dummy_ecommerce_dataset = get_ecommerce_1_dataset
-get_sample_ecommerce_dataset = get_ecommerce_2_dataset
-get_ecommerce_dataset = get_ecommerce_3_dataset
+### For backwards compatability
```
Comment (charyeezy) [inline relevanceai/datasets.py:476]: @boba-and-beer  Hello! I have resolved this here, however if you want me instead to keep fn name as `get_ecommerce_1_dataset`, `get_ecommerce_2_dataset` etc pls let me know!

File: relevanceai/datasets.py [hunk 2]
```diff
@@ -480,6 +473,13 @@ def get_machine_learning_research_dataset():
     raise NotImplementedError
 
 
-get_dummy_ecommerce_dataset = get_ecommerce_1_dataset
-get_sample_ecommerce_dataset = get_ecommerce_2_dataset
-get_ecommerce_dataset = get_ecommerce_3_dataset
+### For backwards compatability
+
+get_ecommerce_1_dataset = get_ecommerce_dataset_encoded
```
Comment (boba-and-beer) [inline relevanceai/datasets.py:478]: I think we should make full use of Python's multiple assignment property as per the comment as I don't see a need to assign it again as it reduces maintainability in future as we are less sure about what's equal.

Comment (charyeezy) [reply relevanceai/datasets.py]: Ahh I understand your meaning of course!

**Review by charyeezy — COMMENTED** [review]
Resolved

**Pull request merged**